common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -123,7 +123,7 @@ def wait_for_status(self, status):
         self.problem_page.wait_for_element_visibility(selector, msg)
 
     @abstractmethod
-    def answer_problem(self, correct):
+    def answer_problem(self, correctness):
```

Comment:
Note: I updated all the answer_problem methods to handle the partially-correct answer type.